cfpb/cfgov-refresh
# Mega Menu: Lay out menu overview links as table display cells

## Changes

- Makes the menu overview links at desktop table display cells, so they are responsive across all desktop sizes.

## Testing

1. Pull branch and build.
2. Resize the browser window and see that the menu tabs evenly distribute themselves across desktop screen sizes.

## Screenshots

![menu-responsive](https://user-images.githubusercontent.com/704760/78406371-7d0ef100-75d0-11ea-8eeb-09ca66d09bcc.gif)
